edenia/antelope-tools
# Feat/add eos rating 666

### Include the average rating of a BP

### What does this PR do?

- Resolve #666 
- Add route and query to get EOSRate ratings from the API
- Refactor code of producerInformation component

### Steps to test

1. Run the project locally.
1. Go to "Block producers" page
1. Check the EOSRate value in the stats of a BP

### Screenshot

![imagen](https://user-images.githubusercontent.com/66583677/208938082-98f42920-68c0-4b86-b438-7d40506ed5cc.png)
